pkg/agent/agent_windows.go

```diff
@@ -32,3 +39,138 @@ func (i *Initializer) setupExternalConnectivity() error {
 	}
 	return nil
 }
+
+// prepareHostNetwork creates HNS Network for containers.
+func (i *Initializer) prepareHostNetwork() error {
+	// If the HNS Network already exists, return immediately.
+	hnsNetwork, err := hcsshim.GetHNSNetworkByName(util.LocalHNSNetwork)
+	if err == nil {
+		// Save the uplink adapter name to check if the OVS uplink port has been created in prepareOVSBridge stage.
+		i.nodeConfig.UplinkNetConfig.Name = hnsNetwork.NetworkAdapterName
+		return nil
+	}
+	if _, ok := err.(hcsshim.NetworkNotFoundError); !ok {
+		return err
+	}
+	// Get uplink network configuration.
+	_, adapter, err := util.GetIPNetDeviceFromIP(i.nodeConfig.NodeIPAddr.IP)
+	if err != nil {
+		return err
+	}
+	i.nodeConfig.UplinkNetConfig.Name = adapter.Name
+	i.nodeConfig.UplinkNetConfig.MAC = adapter.HardwareAddr
+	i.nodeConfig.UplinkNetConfig.IP = i.nodeConfig.NodeIPAddr
+	i.nodeConfig.UplinkNetConfig.Index = adapter.Index
+	defaultGW, err := util.GetDefaultGatewayByInterfaceIndex(adapter.Index)
+	if err != nil {
+		return err
+	}
+	i.nodeConfig.UplinkNetConfig.Gateway = defaultGW
+	dnsServers, err := util.GetDNServersByInterfaceIndex(adapter.Index)
+	if err != nil {
+		return err
+	}
+	i.nodeConfig.UplinkNetConfig.DNSServers = dnsServers
+	// Create HNS network.
+	return util.PrepareHNSNetwork(i.nodeConfig.PodCIDR, i.nodeConfig.NodeIPAddr, adapter)
+}
+
+// cleanupHostNetwork deletes HNS Network created by Antrea.
+func (i *Initializer) cleanupHostNetwork() error {
```

Comment:
This one can be removed.

Reply:
Thanks Jianjun, remove this function in linux and windows files.